content/go.md

```diff
@@ -39,109 +40,151 @@ $ go get go.opencensus.io
 
 See the [tag](https://godoc.org/go.opencensus.io/tag), [stats](https://godoc.org/go.opencensus.io/stats), [trace](https://godoc.org/go.opencensus.io/trace) godoc for the API reference and [examples](https://github.com/census-instrumentation/opencensus-go/tree/master/examples) directory for samples.  
 
-**Example**  
 
-The following example uses  demonstrates how to record stats and traces for a video processing system. It records data with the “frontend” dimension to be able to break down collected data by the frontend user started the video processing from.
+---
+
+## Tags
+
+Tags represent propagated key-value pairs. They are propagated using `context.Context`
+in the same process or can be encoded to be transmitted on the wire.
+
+Package tag allows adding or modifying tags in the current context.
 
+```go
+ctx, err = tag.New(ctx,
+	tag.Insert(osKey, "macOS-10.12.5"),
+	tag.Upsert(userIDKey, "cde36753ed"),
+)
+if err != nil {
+	log.Fatal(err)
+}
 ```
-$ cd $(go env GOPATH)/src/go.opencensus.io/examples/helloworld
-$ go get -v . # get the dependencies
-$ go run main.go
+
+---
+
+## Stats
+
+OpenCensus stats collection happens in two stages:
+
+* Definition of measures and recording of data points
+* Definition of views and aggregation of the recorded data
+
+### Recording
+
+Measurements are data points associated with a measure.
+Recording implicitly tags the set of Measurements with the tags from the
```

Comment:
This sentence is hard to read, how about "Recording the set of Measurements with implicit tags from the provided context"?